docs/operator-dependencies-and-requirements.md

```diff
@@ -0,0 +1,40 @@
+# Operator Dependency and Requirement Resolution
+
+## Overview on Dependency Principles
+
+OLM manages the dependency resolution and upgrade lifecycle of running operators.
+
+An operator's dependencies are defined through the (Group, Version, Kind) of of an API. An operator cannot define a dependency on a specific version (e.g. `etcd-operator v0.9.0`) or instance (e.g. `etcd v3.2.1`) of an operator. This encourages operator authors to depend on the interface and not the implementation; thereby allowing operators to update on individual release cycles.
```

Comment:
Double "of" in "(Group, Version, Kind) of of an API"

Reply:
Addressed.